packages/@uppy/core/src/Uppy.js

```diff
@@ -181,8 +181,8 @@ class Uppy {
   /**
    * Shorthand to set state for a specific file.
    */
-  setFileState (fileID, state) {
-    if (!this.getState().files[fileID]) {
+  setFileState (fileID, state, checkIfFileExist = true) {
```

Comment:
Personally I'd take a single extra negligible computation over adding a new parameter to a function. Will there every be a case outside this? Is it worth making a function more complex?